src/battle_script_commands.c

```diff
@@ -4686,7 +4686,8 @@ static void MoveValuesCleanUp(void)
     gIsCriticalHit = FALSE;
     gBattleScripting.moveEffect = 0;
     gBattleCommunication[MISS_TYPE] = 0;
-    gHitMarker &= ~HITMARKER_DESTINYBOND;
+    if (!gMultiHitCounter)
```

Comment:
Potential improvement for the future: `if (gMultiHitCounter > 1)`, as setting multi-hit to 1 could potentially make Destiny Bond not work here.

Reply:
> unless it's reduced before it reaches this?
Yes, it has to be reduced to zero. Otherwise it doesn't work

Reply:
but `gMultiHitCounter > 0` would still look better imo

Reply:
Understood :)

Reply:
let's keep `if (!gMultiHitCounter)` for consistency. It is used in a lot of places like that.